TuneUp/TuneUpViewExtension.cs

```diff
@@ -44,17 +44,26 @@ public override void Loaded(ViewLoadedParams p)
                 if (TuneUpMenuItem.IsChecked)
                 {
                     p.AddToExtensionsSideBar(this, TuneUpView);
-                    ViewModel.EnableProfiling();
+                    ViewModel.SwitchToManualMode();
+                    ViewModel.EnableProfiling();                    
                 }
                 else
                 {
                     p.CloseExtensioninInSideBar(this);
                 }
             };
 
+            // Bind the IsChecked property to the IsTuneUpActive property
+            TuneUpMenuItem.Checked += (sender, args) => ViewModel.IsTuneUpChecked = true;
+            TuneUpMenuItem.Unchecked += (sender, args) => ViewModel.IsTuneUpChecked = false;
+
             // Add this view extension's menu item to the Extensions tab or View tab accordingly.
             var dynamoMenuItems = p.dynamoMenu.Items.OfType<MenuItem>();
-            var extensionsMenuItem = dynamoMenuItems.Where(item => item.Header.ToString() == "_Extensions");
+            // TODO: Investigate why TuneUpMenuItem is not added to the specified MenuItem
+            // When _Extensions is specified, TuneUpMenuItem goes to the View menu in Dynamo 3.0-
+            // and does not appear in Dynamo 3.1+.
+            // We need to specify _View for TuneUpMenuItem to be added to the Extensions menu
+            var extensionsMenuItem = dynamoMenuItems.Where(item => item.Header.ToString() == "_View");
```

Comment:
Confused by this a bit.. we specify `_View` to be added to Extentions munue?

Reply:
You and me, brother 😁 
I could not figure out why that is, might have to do with the way Dynamo loads the menus. Looking and the old master seem like this hasn't worked before either. In old demos Show TuneUp is always called from the View menu although the code was specifying `_Extensions`. I thought this is decent workaround for the time being as it's not a part of the ticket.

Reply:
Two things: 
1. Thank you Ivo for shifting it to Extensions. This is 100% where it should live!
2. We should make sure the coat is hardened, so if there is a proper way to do it please refactor :)